src/dfx/src/commands/canister/call.rs

```diff
@@ -73,12 +73,12 @@ struct CallIn {
     method_name: String,
     #[serde(with = "serde_bytes")]
     args: Vec<u8>,
-    cycles: u64,
+    cycles: u128,
 }
 
-async fn do_wallet_call(wallet: &Canister<'_, Wallet>, args: &CallIn) -> DfxResult<Vec<u8>> {
+async fn do_wallet_call(wallet: &WalletCanister<'_>, args: &CallIn) -> DfxResult<Vec<u8>> {
     let (result,): (Result<CallResult, String>,) = wallet
-        .update_("wallet_call")
+        .update_("wallet_call128")
```

Comment:
Should this go through CallForwarder so that the agent can call either `wallet_call` or `wallet_call128` depending on the wallet version?